conf/messages.en-NZ

```diff
@@ -28,7 +28,7 @@ audit.ribbon.missing.ramp = <u>M</u>issing Drop Kerb
 audit.ribbon.no.sidewalk = <u>N</u>o Footpath
 audit.ribbon.crosswalk = Pedestrian<br>Crossing (<u>W</u>)
 audit.ribbon.occlusion = Can''t See the Footpath (<u>B</u>)
-audit.left.ui.jump.explore.title = Go to a new location or a new neighbourhood?
+audit.left.ui.jump.explore.title = If you can''t move forward, click the "Stuck" button instead!
```

Comment:
You can actually fully remove this translation! The only reason we had it was because of the different spelling of neighborhood/neighbourhood. If you remove it, it will default back to `messages.en`!